### DISCUSSION TITLE
`sort-keys` rule

### DISCUSSION THREAD

**Discussion by dalechyn** [biomejs/biome#2079] [Ideas]
Ref: https://github.com/biomejs/biome/discussions/3#discussioncomment-7489793

It would be awesome to have object properties alphabetic sorting in place!

**Comment by arendjr** [comment]
Please keep the discussion centered in #3 . You can add upvotes right there.

**Comment by PaulRBerg** [comment]
This is now available in Biome v2 (currently in beta):

https://next.biomejs.dev/assist/actions/use-sorted-keys/
